File: hafta3/sum.java
import java.util.Scanner;

public class sum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] numbers = new int[5];
        int sum = 0;

        // Kullanıcıdan 5 sayı alma
        for (int i = 0; i < 5; i++) {
            System.out.print((i + 1) + ". sayıyı girin: ");
            numbers[i] = scanner.nextInt();
            sum += numbers[i];
        }

        // Ortalama hesaplama
        double average = sum / 5.0;

        // Sonuçları yazdırma
        System.out.println("Toplam: " + sum);
        System.out.println("Ortalama: " + average);

        scanner.close();
    }
}

File: hafta3/sum.c
#include <stdio.h>

int main() {
    int numbers[5];
    int sum = 0;

    // Kullanıcıdan 5 sayi almak
    for (int i = 0; i < 5; i++) {
        printf("%d. sayiyi girin: ", i + 1);
        scanf("%d", &numbers[i]);
        sum += numbers[i];
    }

    // Toplami yazdir
    printf("Toplam: %d\n", sum);

    return 0;
}
